src/VirtoCommerce.NotificationsModule.Core/Services/INotificationLayoutSearchService.cs

```diff
@@ -0,0 +1,9 @@
+namespace VirtoCommerce.NotificationsModule.Core.Services
+{
+    /// <summary>
+    /// The service is for to make a query to Database and get a list of notification layouts
+    /// </summary>
+    public interface INotificationLayoutSearchService
```

Comment:
INotificationLayoutSearchService should be inherited from ISearchService<...> Ask @artem-dudarev